index.html

```diff
@@ -57,6 +59,169 @@ <h1 id="home-section-title">I am Hiro Glad to See You!</h1>
           </div>
         </div>
       </section>
+
+      <section class="work-section">
+        <div class="card-grid">
+          <div class="work-section-card">
+            <div class="img-container">
+              <img src="/img/SnapshootPortfolio.png" alt="Work" />
```

Comment:
- [ ] Your images appear broken. To fix this, instead of `/img/picture.png`, use `./img/picture.png`. Kindly fix __all__ the broken images